pkgs/by-name/ca/cargo-geiger/package.nix

```diff
@@ -20,7 +21,16 @@ rustPlatform.buildRustPackage rec {
     rev = "cargo-geiger@v${version}";
     hash = "sha256-/5yuayqneZV6aVQ6YFgqNS2XY3W6yETRQ0kE5ovc7p8=";
   };
-  cargoHash = "sha256-lhojo3dhsM9y1SxpVMH93yv+JeNfTL7VLsbTp9ErgIQ=";
+
+  cargoPatches = [
+    (fetchpatch2 {
+      name = "fix-build-with-rust-1.80";
+      url = "https://github.com/geiger-rs/cargo-geiger/pull/528.patch";
```

Comment:
This is not a stable URL.  If the PR content changes, the hash will break.  For unmerged changes, please just put the patch in Nixpkgs — even using a commit URL isn't good because if the PR changes, the old commit will vanish once GitHub GCs, breaking the URL.

Reply:
Thanks, good to know about GitHub GCs. I totally believed that deleted commits are publicly accessible for a long time as I could reach them after deleting.